packages/data-broker/src/services/dhis/getDhisApi.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
```

Comment:
Love this set of functions being encapsulated in a single file! Really helps make clear the different ways the interface is called!